GOVERNANCE.md

```diff
@@ -1,18 +1,97 @@
 ---
 # SPDX-License-Identifier: CC0-1.0
-# SPDX-FileCopyrightText: 2019-2024 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS
+# SPDX-FileCopyrightText: 2024 Standard for Public Code Authors, https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/AUTHORS.md
 ---
-# Governance
 
-This standard lies at the core of the codebase stewardship provided by the [Foundation for Public Code](https://publiccode.net/).
-We decide if a codebase is ready for community co-development based on this document.
+# Governance.md
 
-The standard is maintained by Foundation for Public Code staff.
+The Standard for Public Code is a community governed project.
 
-[We welcome contributions, such as suggestions for changes or general feedback, from anyone.](/CONTRIBUTING.md)
+## Principles
 
-Because of the important role that the Standard for Public Code has in our core process we require the highest standards from the Standard.
+The Standard for Public Code community adheres to the following principles:
 
-We will try to respond promptly to all pull requests.
-The pull request is an opportunity to work together to improve our methods and the Standard.
-We may not accept all changes, but we will explain our logic.
+* Truly open - as little restrictions as possible for anyone to adapt the Standard for Public Code to their context
+* Welcoming and respectful - as a community we want to make it easier for new users to become contributors.
+* Transparent and accessible, changes to the governance, Standard for Public Code code repository, and Standard for Public Code related activities (e.g. level, involvement, etc) are done in public
+* Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles
```

Comment:
I don't know how to define “technical merit”.

```suggestion
* Ideas and contributions are accepted according to their alignment with project objectives, scope, and design principles
```

Reply:
Implemented in [be0991b](https://github.com/standard-for-public-code/standard-for-public-code/pull/1023/commits/be0991b49ba1fa77861d9b8c9029acb03aa14a10)